plugins/sitegraph/pom.xml

```diff
@@ -151,7 +151,6 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>rat-maven-plugin</artifactId>
-                <version>1.0-alpha-2</version>
```

Comment:
Same here

```
<groupId>org.apache.rat</groupId>
<artifactId>apache-rat-plugin</artifactId>
```

Reply:
Successfully migrated to apache-rat-plugin.